Add new sections and articles (#244)
AdguardTeam/KnowledgeBaseDNS: docs/private-dns/connect-devices/mobile-and-desktop/ios.md

---
title: iOS
sidebar_position: 3
---

> To connect an iOs device to AdGuard DNS, first add it to Dashboard:

Reviewer: ```suggestion
To connect an iOS device to AdGuard DNS, first add it to *Dashboard*:
```

1. smth
1. smth
1. smth

## Use AdGuard Ad Blocker (paid option)

## Use AdGuard VPN

## Configure Private DNS manually

## Configure plain DNS

> If you prefer not to use extra software for DNS configuration, you can opt for unencrypted DNS. You have two choices: using linked IPs or dedicated IPs.

Reviewer: ```suggestion
If you prefer not to use extra software to configure DNS, you can opt for unencrypted DNS. There are two options: using linked IPs or dedicated IPs.
```

1. Install the [AdGuard VPN app](https://adguard-vpn.com/ios/overview.html) on the device you want to connect to AdGuard DNS.
1. Open the AdGuard VPN app.
1. Select the menu by clicking the gear icon in the lower right corner of the screen.

    ![Gear icon *mobile](https://cdn.adtidy.org/content/kb/dns/private/new_dns/connect/ios_vpn/ios_step3.jpg)

1. Open *General*.

    ![General settings *mobile](https://cdn.adtidy.org/content/kb/dns/private/new_dns/connect/ios_vpn/ios_step4.jpg)

1. Select *DNS server*.

    ![DNS server *mobile](https://cdn.adtidy.org/content/kb/dns/private/new_dns/connect/ios_vpn/ios_step5.png)

1. Scroll down to *Add custom DNS server*.

    ![Add server *mobile](https://cdn.adtidy.org/content/kb/dns/private/new_dns/connect/ios_vpn/ios_step6.png)

1. Copy one of the following DNS addresses and paste it into the *DNS server addresses* text field. If you are not sure which one to prefer, select *DNS-over-HTTPS*.

    ![DoH server *mobile](https://cdn.adtidy.org/content/kb/dns/private/new_dns/connect/ios_vpn/ios_step7_1.png)

    ![Custom DNS server *mobile](https://cdn.adtidy.org/content/kb/dns/private/new_dns/connect/ios_vpn/ios_step7_2.jpg)

> 1. Tap *Save*.

Author: сделала тэпы
